lib/app.js

```diff
@@ -44,6 +45,30 @@ class App {
                 res.setHeader('Location', location);
             }
 
+            // send file function
+            res.sendFile = (dir, file) => {
+              if(!dir) dir = ecstatic;
+              readFile(dir, file, (err, str) => {
+                if(!err && str) {
+                    
+                  // check if the is an html file
+                  if(file.indexOf('.html') > -1 || file.indexOf('.htm') > -1) {
+                    res.setHeader('Content-type', 'text/html');
+                  } else if(file.indexOf('.pdf') > -1) {
+                    res.setHeader('Content-type', 'application/pdf');
+                  }
+
+                  res.send(str);
+                  res.end();
+                } else {
+                  // console.log('Error from read file', err);
+                  res.send(err);
+                  res.end();
```

Comment:
Don't end the response here, these methods are just responsible for sending data and closing the connection should be up to user and they do in router code they do